### ISSUE TITLE
Doesn't work Go to definition by Ctrl+Click(Linux)/Cmd+Click(Mac)

### ISSUE THREAD

**Issue by mikhalenia** [Komodo/KomodoEdit#3240]
### Short Summary
When I press Ctrl(Linux)/Cmd(Mac) and Click on some variable/function/class in Komodo IDE 10 it goes to definition but it doesn't work on Komodo IDE 11.

### Steps to Reproduce
1. Open Komodo IDE 11
2. Create simple PHP/JS file which contents variable, class and function ([PHP](https://kopy.io/h3lMQ), [JS](https://kopy.io/gw6Nd))
3. Focus on variable/class/function press Ctrl/Cmd and Click on it

### Expected results
I think It has to go to definition of variable/class/function like in Komodo IDE 10.

### Actual results
It sets additional cursor and that's it.

### Platform Information
Komodo IDE 11
It's reproduced on Ubuntu 15.10 and Mac OS Sierra 10.12.6

### Additional Information
I've got two versions of Komodo IDE on my computer at the same time: Komodo IDE 10 and Komodo IDE 11

**Comment by mitchell-as** [comment]
I am able to reproduce this on a PHP file:

```
<?php

function foo() {

}

foo()
```

Ctrl+Clicking the `foo` on the bottom line does not jump to the function definition. Right-click > Go to Definition does jump to the function definition.

**Comment by cgchoffman** [comment]
This isn't a bug from what I understand.  `Ctrl + click` was not implemented for go to definition due to that function being overloaded.  Just changing to enhancement as it's open for debate whether it gets implemented again.

**Comment by Naatan** [comment]
Correct, this isn't a bug. It is by design. Too much was living under Ctrl+Click, and right click > go to anything isnt much work to do instead.
